Scripts/Mac/BuildGVFSForMac.sh

```diff
@@ -59,9 +61,12 @@ xcodebuild -configuration $CONFIGURATION -workspace $NATIVEDIR/GVFS.Native.Mac.x
 USERNOTIFICATIONDIR=$VFS_SRCDIR/GVFS/GVFS.Notifications/VFSForGit.Mac
 USERNOTIFICATIONPROJECT="$USERNOTIFICATIONDIR/VFSForGit.xcodeproj"
 USERNOTIFICATIONSCHEME="VFS For Git"
-updateAppVersionCmd="(cd \"$USERNOTIFICATIONDIR\" && /usr/bin/xcrun agvtool new-marketing-version \"$VERSION\")"
-echo $updateAppVersionCmd
-eval $updateAppVersionCmd
+# don't update version number for developer builds
+if [ "$runVersionUpdater" == "yes" ]; then
+    updateAppVersionCmd="(cd \"$USERNOTIFICATIONDIR\" && /usr/bin/xcrun agvtool new-marketing-version \"$VERSION\")"
+    echo $updateAppVersionCmd
+    eval $updateAppVersionCmd
```

Comment:
Do we need to check the return code of this command (and exit on failure?)

Reply:
added an exit on error.